pages/common/enscript.md

```diff
@@ -0,0 +1,23 @@
+# enscript
+
+> A tool to convert text files to PostScript, HTML, RTF, ANSI, and overstrikes.
+
+- Generate PostScript from a file and output to another:
+
+`enscript {{text_file}} --output={{output_file}}`
+
+- Generate a certain output language (eg. _html_) from a file and output to another:
+
+`enscript {{text_file}} --language={{language}} --output={{output_file}}`
+
+- Generate PostScript from a file and output to another with 1 to 9 coloums per page in landscape:
+
+`enscript {{text_file}} --columns={{num}} --landscape --output={{output_file}}`
+
+- Display avaliable syntax highlighting:
+
+`enscript --help-highlight`
+
+- Genereate PostScript from a file and output to another with syntax highlighting and color for a specified language:
+
+`enscript {{text_file}} --color=1 --highlight={{language}} --output={{output_file}}`
```

Comment:
Is `1` the equivalent to `true`? Or is `1` a specific colour scheme?

Reply:
I'm not sure, I'll double check after school. Sorry for all the mistakes.

Reply:
No need to be sorry! Thank you for the page. 👍

Reply:
I had some time in the morning to experiment. According to the man pages, there is only one colour scheme, it appears to only works with syntax highlighting, and using `true` instead of `1` leaves it colourless. But I will push my other fixes.

Reply:
Sounds good to me 👍